Sparkle/SUAutomaticUpdateAlert.m

```diff
@@ -88,4 +89,30 @@ - (NSString *)descriptionText
     }
 }
 
+- (NSTouchBar *)makeTouchBar
+{
+    NSButton *installButton, *laterButton, *cancelButton;
+    for (NSView *control in self.window.contentView.subviews) {
+        if ([control isKindOfClass:[NSButton class]]) {
+            NSButton *button = (NSButton *)control;
+            if (button.action == @selector(installNow:)) {
+                installButton = button;
```

Comment:
I did use outlets in my initial draft, but I was horrified by the xibs...
So I decided to use a 'just working' way by iterating through subviews, and change them to outlets in the future.

I think I can update them later.